src/Migrate/custom/Start-AzMigrateTestMigration.ps1

```diff
@@ -131,7 +131,10 @@ function Start-AzMigrateTestMigration {
             Import-module -Name Az.Compute
             Import-module -Name Az.Network
             $AvSetName = $AvSetId.Split("/")[-1];
-            $AvSetRg = $AvSetId.Split("/")[-5];
+            $AvSetRg = $AvSetId.Split("/")[-5];            
+            $TargetSubscriptionId = $AvSetId.Split("/")[-7];
+            $SourceSubscriptionId = (Get-AzContext -ErrorVariable notPresent -ErrorAction SilentlyContinue).Subscription.Id
+            Set-AzContext -SubscriptionId $TargetSubscriptionId -ErrorVariable notPresent -ErrorAction SilentlyContinue
             $AvSet = Get-AzAvailabilitySet -ResourceGroupName $AvSetRg -Name $AvSetName -ErrorVariable notPresent -ErrorAction SilentlyContinue
```

Comment:
>AzAvailabilitySet [](http://example.com/codeflow?start=25&length=17)

can we not use the Get-AzResource to get?

Reply:
We are fetching VM details from Avset, which we can't do using Get-AzResource